packages/slice-machine/src/models/slice/reducer.ts

```diff
@@ -202,16 +203,34 @@ export function reducer(
   return {
     ...result,
     isTouched: (() => {
+      // If the model is not pushed the model cannot be touched
+      if (result.__status === LibStatus.NewSlice) {
+        return false;
+      }
+
       return (
         !equal(result.initialVariations, result.variations) ||
         !equal(result.initialMockConfig, result.mockConfig)
       );
     })(),
     __status: (() => {
-      return !equal(result.screenshotUrls, result.initialScreenshotUrls) ||
-        !compareVariations(result.remoteVariations, result.initialVariations)
+      // If the model is not pushed the only status possible is new slice
+      if (result.__status === LibStatus.NewSlice) {
```

Comment:
On the ComponentUI there is a function to compute the status, we should reuse it here to keep the logic only once.